packages/launchpad/src/settings/device/ProxySettings.spec.tsx

```diff
@@ -2,6 +2,11 @@ import ProxySettings from './ProxySettings.vue'
 
 describe('<ProxySettings />', () => {
   it('renders', () => {
-    cy.mount(() => <ProxySettings />)
+    cy.viewport(400, 400)
+    .mount(() => <ProxySettings />).get('body')
+    .findByText('Proxy Server').get('body')
+    .findByText('Proxy Bypass List')
+    .get('[data-testid=bypass-list]').should('be.visible')
```

Comment:
Could use `findByTestId`: https://testing-library.com/docs/queries/bytestid

Reply:
Hm. It wasn't working for me at the time. I also don't know if I like testing library's need to scope the next query to the return value of the command before it... e.g. `.get('body')` between tests.